<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>861</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Send_pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>800</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>851</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>830</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>在线人数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ClientNumber">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>830</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="usernametable">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>100</y>
     <width>256</width>
     <height>681</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="jiacu">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>jiacu</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>jiachu.jpg</normaloff>jiachu.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="qingxie">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>qingxie</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/qinxie.jpg</normaloff>:/new/prefix1/qinxie.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
